Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gradle:7.4-jdk8-focal AS build
+WORKDIR /notification-plugin
+COPY . ./
+RUN gradle clean classes
```

Comment:
Why do you copy the whole project into the container and start it in the CI with `gradle run`? I imagine this introduces quite a bit of overhead. Would it be possible to build a runnable JAR instead and then in the CI script just run `java -jar notification-plugin.jar`?
That would also allow for a smaller container with just the JRE and the JAR.

For the first implementation the current running is probably fine but if you think it would be possible to optimize it with the JAR, please open an issue so that it won’t be forgotten for a follow-up.

Reply:
Yes, I agree. For the first version, our goal is to develop a simple prototype relative quickly.
Anyway, this is definitely a good point, we have to address in the future.

Reply:
Just created the issue #2